### DISCUSSION TITLE
WEBDAV mount TV SHows

### DISCUSSION THREAD

**Discussion by lstryhrn** [itsToggle/plex_debrid#346] [General]
So I am using the wevdav remotes because the forks just makes my plex crash with the scan. It also makes explorer crash as well.

The instruction below does not work with tv shows.  The movies work great.

Movie/Show sorting for WebDAV remotes:
Using rclone filtering, you can split your WebDAV remote into movie and show content.
start one instance of rclone by adding the flag --exclude "**{{(s|S)([0-9]+)( |.|e|E)}}**" to your mount command, mount to a folder named "movies"
start another instance of rclone by adding the flag --include "**{{(s|S)([0-9]+)( |.|e|E)}}**" to your mount command, mount to a folder named "shows"

**Comment by itsToggle** [comment]
crashing is not normal. File a bug report, elaborate on your issue.

Could you explain what is not working for tv shows?

**Reply by lstryhrn** [reply]
Thanks I will file a crash report. So anytime you use a --include it doesnt exlcude movies. The --exclude works fine. With the webdav you have two folders torrents and links. Everything is in the torrent folder. The tv shows are not being separated.
